cypress/e2e/article.cy.js

```diff
@@ -1,24 +1,74 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
+import ArticleEditorPageObject from '../support/pages/articleEditor.pageObject';
+import ArticlePageObject from '../support/pages/articlePage.pageObject';
+import HomePageObject from '../support/pages/home.pageObject';
+
+const articleEditorPage = new ArticleEditorPageObject();
+const articlePage = new ArticlePageObject();
+const homePage = new HomePageObject();
+
 describe('Article', () => {
-  before(() => {
+  let user;
+  let article;
+
+  const articleChaned = {
+    title: faker.lorem.words(),
+    description: faker.lorem.words(),
+    body: faker.internet.email()
+  };
 
+  before(() => {
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
+    cy.task('generateArticle').then((generateArticle) => {
+      article = generateArticle;
+    });
   });
 
   beforeEach(() => {
     cy.task('db:clear');
+    homePage.visit();
```

Comment:
In the 'beforeEach' block, you are visiting the HomePage, logging in and then in each test visiting the ArticleEditorPage again. You can avoid visiting HomePage in the 'beforeEach' block if it's not necessary for each test.